javascript/low-http-server/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  github: jkyberneees/low-http-server
+  version: 2.1
```

Comment:
you need to add

```yaml
build_deps:
  - git

bin_deps:
  - gcompat # required by uWebSocket
```
@jkyberneees 

to ensure compatibility with `alpine` **containers**